openedx/features/course_experience/static/course_experience/js/CourseSock.js

```diff
@@ -69,8 +69,9 @@ export class CourseSock {  // eslint-disable-line import/prefer-default-export
 
     $upgradeToVerifiedButton.on('click', () => {
       Logger.log(
-        'User clicked the upgrade button in the verification sock.',
+        'edx.course.upgrade.clicked',
```

Comment:
@dkh The existing event is "edx.course.enrollment.upgrade.clicked" and your new event is 'edx.course.upgrade.clicked'.  Is our plan to rename the event, is this a different event, or do we need to use the existing name?  Assuming we want one event and not two, we should probably add a location to the existing events as well.
FYI: @catong @marcotuts